igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/client/model/Commit.groovy

```diff
@@ -37,7 +37,24 @@ class Commit {
     @SerializedName("author_name")
     String authorName
 
+    @SerializedName("compare_url")
+    String compareUrl
```

Comment:
I think travis is github specific. https://docs.travis-ci.com/user/customizing-the-build/#What-git-Repository-Providers-can-I-use